content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -210,6 +210,7 @@ currently include:
  * `node.kubernetes.io/memory-pressure`: Node has memory pressure.
```

Comment:
In L200: "Kubernetes 1.6 has alpha support for representing node problems."

Is it still alpha support?

Reply:
It seems that we don't have alpha support taints anymore. 
"... Kubernetes 1.6 has alpha support for representing node problems" --> "... Kubernetes taints nodes that have problems."

Reply:
"Taint based Evictions (alpha feature)" in L186 needs to be updated as well.

Reply:
`TaintBasedEviction` is still alpha :( , I'm trying to graduate it to beta in one or two release.